capirca/lib/arista_tp.py

```diff
@@ -0,0 +1,1036 @@
+                        term = copy.deepcopy(pol_term)
+                    else:
+                        term = pol_term
+
+                    term_type = ft
+                    term.name = self.FixTermLength(term.name)
+                    # TODO(sulrich): if term names become unique to address
+                    # families, this can be removed.
+                    if (filter_type == "mixed" and term_type == "inet6"):
+                        term.name = term.name + "_v6"
+
+                    if term.name in term_names:
+                        raise AristaTpDuplicateTermError(
+                            "multiple terms named: %s" % term.name
+                        )
+                    term_names.add(term.name)
+
+                    term = self.FixHighPorts(term, af=term_type)
+                    if not term:
+                        continue
+
+                    if term.expiration:
+                        if term.expiration <= exp_info_date:
+                            logging.info(
+                                "INFO: term %s in policy %s expires "
+                                "in less than two weeks.",
+                                term.name,
+                                filter_name,
+                            )
+                        if term.expiration <= current_date:
+                            logging.warning(
+                                "WARNING: term %s in policy %s is expired and "
+                                "will not be rendered.",
+                                term.name,
+                                filter_name,
+                            )
+                            continue
+
+                    # emit warnings for unsupported options / terms
+                    if term.option:
+                        unsupported_opts = []
+                        for opt in [str(x) for x in term.option]:
+                            if opt.startswith("sample") or \
+                               opt.startswith("first-fragment"):
+                                unsupported_opts.append(opt)
+
+                        # unsupported options are in use and should be skipped
+                        if len(unsupported_opts) > 0:
+                            logging.warning(
+                                "WARNING: term %s in policy %s uses an "
+                                "unsupported option (%s) and will not be "
+                                "rendered.",
+                                term.name,
+                                filter_name,
+                                " ".join(unsupported_opts),
+                            )
+                            continue
+
+                    has_unsupported_match_criteria = (
+                        term.dscp_except or
+                        term.dscp_match or
+                        term.ether_type or
+                        term.flexible_match_range or
+                        term.forwarding_class or
```

Comment:
Would forward_class be what could implement traffic-policy's traffic class? Not sure if this is super urgent, or a must-have kinda feature, but I saw that traffic classes are supported. I believe that means the same as forwarding classes, except in Juniper, they're BE, AF etc, and in Arista they are 0-7.